src/sudo/mod.rs

```diff
@@ -75,9 +74,7 @@ fn sudo_process() -> Result<(), Error> {
     self_check()?;
     kernel_check()?;
 
-    let pipeline = Pipeline {
-        authenticator: PamAuthenticator::new_cli(),
-    };
+    let pipeline = Pipeline {};
```

Comment:
Does this object still have a purpose after this refactor?

Reply:
Nope, removed.